.gitignore

```diff
@@ -14,8 +14,10 @@ yarn-error.log
 # macOS
 .DS_Store
 
+.env.*
```

Comment:
What about having an .env.sample so that devs know which variables should be present? In that case that file should be tracked :)

Reply:
Great catch! I added the .env.sample file.